src/Domain/DomainHelpers.hpp

```diff
@@ -66,9 +66,28 @@ void set_periodic_boundaries(
         neighbors_of_all_blocks);
 
 /// \ingroup ComputationalDomainGroup
-/// These are the six Wedge3Ds used in the DomainCreators for Sphere and Shell.
+/// These are the Wedge3Ds used in the Sphere, Shell, and BBH DomainCreators.
+/// The last argument `x_coord_of_shell_center` is used in the BBH DomainCreator
+/// to specify the x-coordinate of the center of the Shell in the TargetFrame.
+/// This function can also be used to produce the CoordinateMaps
+/// which wrap the Sphere and Shell in a cube made of six Wedges.
+/// When `use_half_wedges` is `true`, the wedges in the +z,-z,+y,-y directions
+/// are cut in half along their xi-axes. The ten CoordinateMaps are used for
+/// the outermost Blocks of the BBH Domain.
```

Comment:
Can I suggest reorganizing these sentences? I found the logical flow hard to follow.

I took a stab at reorganizing here, that (to me) is easier to follow. But it's possible I misunderstood what is happening and wrote incorrect stuff, so feel free to modify or even ignore:
```
/// These are the CoordinateMaps of the Wedge3Ds used in the Sphere, Shell, and
/// BBH DomainCreators.
/// This function can also be used to wrap the Sphere or Shell in a cube made of
/// six Wedges.
/// The argument `x_coord_of_shell_center` specifies a translation of the Shell
/// in the x direction in the TargetFrame. The BBH DomainCreator uses this to set
/// the position of each BH.
/// When the argument `use_half_wedges` is set to `true`, the wedges in the
/// +z,-z,+y,-y directions are cut in half along their xi-axes. The resulting ten
/// CoordinateMaps are used for the outermost Blocks of the BBH Domain.
```

Reply:
I will use your reorganized sentences :+1:  thanks!